config/routes.rb

```diff
@@ -350,12 +350,16 @@
       put '/how_to_verify' => 'how_to_verify#update'
       get '/document_capture' => 'document_capture#show'
       put '/document_capture' => 'document_capture#update'
+      get '/socure/document_capture' => 'socure/document_capture#show'
+      put '/socure/document_capture' => 'socure/document_capture#update'
       # This route is included in SMS messages sent to users who start the IdV hybrid flow. It
       # should be kept short, and should not include underscores ("_").
       get '/documents' => 'hybrid_mobile/entry#show', as: :hybrid_mobile_entry
       get '/hybrid_mobile/document_capture' => 'hybrid_mobile/document_capture#show'
       put '/hybrid_mobile/document_capture' => 'hybrid_mobile/document_capture#update'
       get '/hybrid_mobile/capture_complete' => 'hybrid_mobile/capture_complete#show'
+      get '/hybrid_mobile/socure/document_capture' => 'hybrid_mobile/socure/document_capture#show', as: :hybrid_mobile_socure_document_capture
+      put '/hybrid_mobile/socure/document_capture' => 'hybrid_mobile/socure/document_capture#update'
```

Comment:
Socure only supports GET and POST for the redirect .... could also punt on this til [LG-14007](https://cm-jira.usa.gov/browse/LG-14007) and [LG-14008](https://cm-jira.usa.gov/browse/LG-14008)